PR18 additional edits

ewquon opened the pull request:
Minor edits, mostly notes for me. Also, does it make sense to always take the first time step or take a time average (if timevarying==False)?

ewquon committed:
Add notes for future development

ewquon committed:
Update docstring, add comments re dimensions

ewquon committed:
Fix docstring

DriesAllaerts commented:
The original get_height() function returned the height for the first time step, so to ensure backward compatibility that is what the current implementation returns for timevarying==False. The question whether or not this is meaningful should be addressed by the original author @phawbeck

hawbecker commented:
I was introduced to [wrf-python](https://wrf-python.readthedocs.io/en/latest/) and no longer really plan to use wrf/utils.py - wrf-python has all of these functions built in already and much more (they basically re-wrote all NCL utilities for WRF in python). It is also maintained by UCAR so it should be better documented / coded / useable.

ewquon committed:
Add firsttime option

If False, then return the time-averaged height coordinate (this makes
more sense to me)

ewquon committed:
Replace 'firsttime' kwarg with 'avgheight'

Remove get_avg_height() to avoid confusion and add note about wrf-python
module. Some or all of these functions may be deprecated in the future.

ewquon commented:
Okay, I say that we keep the functions that we've already written that are straightforward (e.g., calculating and unstaggering heights). I wouldn't expect these functionalities in wrf-python to differ or change over time. I'd rather not add an additional dependency, especially one that doesn't `conda install` easily. For the time being, let's focus on getting your tendencies code to work.
